aria/orchestrator/execution_plugin/ctx_proxy/server.py

```diff
@@ -150,43 +150,101 @@ def __exit__(self, *args, **kwargs):
         self.close()
 
 
-def _process_ctx_request(ctx, args):
+def _process_ctx_request(ctx, args): # pylint: disable=too-many-branches,too-many-statements
```

Comment:
Of course we can, but it doesn't seem a very long function at all. This is one of those cases where PyLint is not being helpful.